### ISSUE TITLE
At least one field must have sortable key :true in order for detail row to work

### ISSUE THREAD

**Issue by aceofwings** [bootstrap-vue-next/bootstrap-vue-next#656]
**Describe the bug**
In order to show the detail row at least one field in BTable must have sortable :true

**To Reproduce**
```
<template>
    <b-table :items="items" :fields="fields">
      <template #cell(name)="row">
        <b-button size="sm" @click="row.toggleDetails">
          {{ row.detailsShowing ? 'Hide' : 'Show' }}           {{row.detailsShowing}}
        </b-button>
      </template>
    </b-table>
</template>

export default defineComponent({
  data() {
    return {
       items : [
         { isActive: true, age: 40, name: { first: 'Dickerson', last: 'Macdonald' } },
        ],
        fields : [
          { key: 'name', label: 'Person full name', sortable: true, sortDirection: 'desc' },
          { key: 'age', label: 'Person age' },
        ]
    }
  }
});
```

Remove sortable true and detailrow show function will stop working

**Environment (please complete the following information):**
 - OS: [e.g. Windows 10]
 - Version [e.g. 0.3.3]

**Additional context**

**Comment by devhus** [comment]
Hmmm, it works correctly with me and on Codesandbox.
https://www.loom.com/share/f5ea260df9924dba9cfb79133a55eafe
https://codesandbox.io/s/brave-perlman-kfu27i?file=/src/App.vue

Please give more details like console logs and try to reproduce it on Codesandbox

**Comment by VividLemon** [comment]
Tried example and it seems to be working (details-showing could be undefined, but that's been fixed now)

**Comment by yanghoxom** [comment]
@VividLemon is it fixed in [nuxt: v0.14.10](https://github.com/bootstrap-vue-next/bootstrap-vue-next/releases/tag/nuxt-v0.14.10) ?

**Comment by yanghoxom** [comment]
@VividLemon I using 0.14.10, and it still is `undefined`
![image](https://github.com/bootstrap-vue-next/bootstrap-vue-next/assets/10399540/205edf90-4d91-451c-9026-0738b4454beb)
